src/Sylius/Behat/Client/ApiPlatformStateMachineClientInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Client;
+
+interface ApiPlatformStateMachineClientInterface
+{
+    public function applyTransition(string $resource, string $id, string $transition): void;
```

Comment:
TBH I don't see any difference between `update`, `create` and `applyTransition` in terms of sylius resources from the client perspective. I would add it as the next function to our default client